src/firewall/functions.py

```diff
@@ -311,6 +311,13 @@ def checkIPnMask(ip):
                 return False
     return True
 
+NOPRINT_TRANS_TABLE = {
+    i: None for i in range(0, sys.maxunicode + 1) if not chr(i).isprintable()
+}
+
+def stripCharacters(rule_str, table):
```

Comment:
Let's create a second function called `stripNonPrintable`. Then we don't need to expose NOPRINT_TRANS_TABLE.